{"nbformat":4,"nbformat_minor":0,"metadata":{"colab":{"provenance":[],"gpuType":"T4"},"kernelspec":{"name":"python3","display_name":"Python 3"},"language_info":{"name":"python"},"accelerator":"GPU"},"cells":[{"cell_type":"code","source":["from google.colab import drive\n","drive.mount('/content/drive')"],"metadata":{"id":"hSrZhTbRSQkV","colab":{"base_uri":"https://localhost:8080/"},"executionInfo":{"status":"ok","timestamp":1758661644656,"user_tz":300,"elapsed":15213,"user":{"displayName":"Maverick Espinosa","userId":"01952844037976753706"}},"outputId":"24dcf3cc-e657-4e8b-e3c4-ea68a3853cd8"},"execution_count":3,"outputs":[{"output_type":"stream","name":"stdout","text":["Mounted at /content/drive\n"]}]},{"cell_type":"code","source":["%cd drive/MyDrive/CS598JBR-Team-27 Drive/\n","!pwd\n","!ls"],"metadata":{"colab":{"base_uri":"https://localhost:8080/"},"id":"jgT3AzXVvMjo","executionInfo":{"status":"ok","timestamp":1758661659594,"user_tz":300,"elapsed":720,"user":{"displayName":"Maverick Espinosa","userId":"01952844037976753706"}},"outputId":"3403301a-9914-46e7-bc18-7ccabe438a56"},"execution_count":4,"outputs":[{"output_type":"stream","name":"stdout","text":["/content/drive/MyDrive/CS598JBR-Team-27 Drive\n","/content/drive/MyDrive/CS598JBR-Team-27 Drive\n","CS598JBR-Team-0  CS598JBR-Team-27\n"]}]},{"cell_type":"code","source":["# TODO: Update Your NetIDs in alphabetical order\n","NetIDs = [\"mae10\", \"mns9\", \"snh6\", \"pranavn6\"]\n","NetIDs_str = \"\".join(NetIDs)\n","\n","# TODO: Clone your GitHub repository\n","! git clone https://github.com/Intelligent-CAT-Lab/CS598JBR-Team-0.git\n","%cd CS598JBR-Team-0"],"metadata":{"id":"Z5zrxVEBVfqs","colab":{"base_uri":"https://localhost:8080/"},"executionInfo":{"status":"ok","timestamp":1758648980884,"user_tz":300,"elapsed":214,"user":{"displayName":"Maverick Espinosa","userId":"01952844037976753706"}},"outputId":"c82fa870-e2f4-4a43-d9bd-76bab85d5d10"},"execution_count":null,"outputs":[{"output_type":"stream","name":"stdout","text":["fatal: destination path 'CS598JBR-Team-0' already exists and is not an empty directory.\n","/content/drive/MyDrive/CS598JBR-Team-27 Drive/CS598JBR-Team-0\n"]}]},{"cell_type":"code","source":["!bash -x MP1/setup_dataset.sh\n","\n","# dataset generation\n","# pipe both stdout+stderr into tee which writes to dataset_generation.log\n","! python3 MP1/dataset_generation.py {NetIDs_str} |& tee dataset_generation.log"],"metadata":{"colab":{"base_uri":"https://localhost:8080/"},"id":"x3Vx1rFjVjLB","executionInfo":{"status":"ok","timestamp":1758649004290,"user_tz":300,"elapsed":23405,"user":{"displayName":"Maverick Espinosa","userId":"01952844037976753706"}},"outputId":"fb77b2c5-f479-4ed4-ac52-d1ecba0e91b3"},"execution_count":null,"outputs":[{"output_type":"stream","name":"stdout","text":["+ pip3 install datasets==2.16.1\n","Collecting datasets==2.16.1\n","  Downloading datasets-2.16.1-py3-none-any.whl.metadata (20 kB)\n","Requirement already satisfied: filelock in /usr/local/lib/python3.12/dist-packages (from datasets==2.16.1) (3.19.1)\n","Requirement already satisfied: numpy>=1.17 in /usr/local/lib/python3.12/dist-packages (from datasets==2.16.1) (2.0.2)\n","Requirement already satisfied: pyarrow>=8.0.0 in /usr/local/lib/python3.12/dist-packages (from datasets==2.16.1) (18.1.0)\n","Collecting pyarrow-hotfix (from datasets==2.16.1)\n","  Downloading pyarrow_hotfix-0.7-py3-none-any.whl.metadata (3.6 kB)\n","Collecting dill<0.3.8,>=0.3.0 (from datasets==2.16.1)\n","  Downloading dill-0.3.7-py3-none-any.whl.metadata (9.9 kB)\n","Requirement already satisfied: pandas in /usr/local/lib/python3.12/dist-packages (from datasets==2.16.1) (2.2.2)\n","Requirement already satisfied: requests>=2.19.0 in /usr/local/lib/python3.12/dist-packages (from datasets==2.16.1) (2.32.4)\n","Requirement already satisfied: tqdm>=4.62.1 in /usr/local/lib/python3.12/dist-packages (from datasets==2.16.1) (4.67.1)\n","Requirement already satisfied: xxhash in /usr/local/lib/python3.12/dist-packages (from datasets==2.16.1) (3.5.0)\n","Requirement already satisfied: multiprocess in /usr/local/lib/python3.12/dist-packages (from datasets==2.16.1) (0.70.16)\n","Collecting fsspec<=2023.10.0,>=2023.1.0 (from fsspec[http]<=2023.10.0,>=2023.1.0->datasets==2.16.1)\n","  Downloading fsspec-2023.10.0-py3-none-any.whl.metadata (6.8 kB)\n","Requirement already satisfied: aiohttp in /usr/local/lib/python3.12/dist-packages (from datasets==2.16.1) (3.12.15)\n","Requirement already satisfied: huggingface-hub>=0.19.4 in /usr/local/lib/python3.12/dist-packages (from datasets==2.16.1) (0.35.0)\n","Requirement already satisfied: packaging in /usr/local/lib/python3.12/dist-packages (from datasets==2.16.1) (25.0)\n","Requirement already satisfied: pyyaml>=5.1 in /usr/local/lib/python3.12/dist-packages (from datasets==2.16.1) (6.0.2)\n","Requirement already satisfied: aiohappyeyeballs>=2.5.0 in /usr/local/lib/python3.12/dist-packages (from aiohttp->datasets==2.16.1) (2.6.1)\n","Requirement already satisfied: aiosignal>=1.4.0 in /usr/local/lib/python3.12/dist-packages (from aiohttp->datasets==2.16.1) (1.4.0)\n","Requirement already satisfied: attrs>=17.3.0 in /usr/local/lib/python3.12/dist-packages (from aiohttp->datasets==2.16.1) (25.3.0)\n","Requirement already satisfied: frozenlist>=1.1.1 in /usr/local/lib/python3.12/dist-packages (from aiohttp->datasets==2.16.1) (1.7.0)\n","Requirement already satisfied: multidict<7.0,>=4.5 in /usr/local/lib/python3.12/dist-packages (from aiohttp->datasets==2.16.1) (6.6.4)\n","Requirement already satisfied: propcache>=0.2.0 in /usr/local/lib/python3.12/dist-packages (from aiohttp->datasets==2.16.1) (0.3.2)\n","Requirement already satisfied: yarl<2.0,>=1.17.0 in /usr/local/lib/python3.12/dist-packages (from aiohttp->datasets==2.16.1) (1.20.1)\n","Requirement already satisfied: typing-extensions>=3.7.4.3 in /usr/local/lib/python3.12/dist-packages (from huggingface-hub>=0.19.4->datasets==2.16.1) (4.15.0)\n","Requirement already satisfied: hf-xet<2.0.0,>=1.1.3 in /usr/local/lib/python3.12/dist-packages (from huggingface-hub>=0.19.4->datasets==2.16.1) (1.1.10)\n","Requirement already satisfied: charset_normalizer<4,>=2 in /usr/local/lib/python3.12/dist-packages (from requests>=2.19.0->datasets==2.16.1) (3.4.3)\n","Requirement already satisfied: idna<4,>=2.5 in /usr/local/lib/python3.12/dist-packages (from requests>=2.19.0->datasets==2.16.1) (3.10)\n","Requirement already satisfied: urllib3<3,>=1.21.1 in /usr/local/lib/python3.12/dist-packages (from requests>=2.19.0->datasets==2.16.1) (2.5.0)\n","Requirement already satisfied: certifi>=2017.4.17 in /usr/local/lib/python3.12/dist-packages (from requests>=2.19.0->datasets==2.16.1) (2025.8.3)\n","INFO: pip is looking at multiple versions of multiprocess to determine which version is compatible with other requirements. This could take a while.\n","Collecting multiprocess (from datasets==2.16.1)\n","  Downloading multiprocess-0.70.18-py312-none-any.whl.metadata (7.5 kB)\n","  Downloading multiprocess-0.70.17-py312-none-any.whl.metadata (7.2 kB)\n","  Downloading multiprocess-0.70.15-py311-none-any.whl.metadata (7.2 kB)\n","Requirement already satisfied: python-dateutil>=2.8.2 in /usr/local/lib/python3.12/dist-packages (from pandas->datasets==2.16.1) (2.9.0.post0)\n","Requirement already satisfied: pytz>=2020.1 in /usr/local/lib/python3.12/dist-packages (from pandas->datasets==2.16.1) (2025.2)\n","Requirement already satisfied: tzdata>=2022.7 in /usr/local/lib/python3.12/dist-packages (from pandas->datasets==2.16.1) (2025.2)\n","Requirement already satisfied: six>=1.5 in /usr/local/lib/python3.12/dist-packages (from python-dateutil>=2.8.2->pandas->datasets==2.16.1) (1.17.0)\n","Downloading datasets-2.16.1-py3-none-any.whl (507 kB)\n","\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m507.1/507.1 kB\u001b[0m \u001b[31m10.2 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n","\u001b[?25hDownloading dill-0.3.7-py3-none-any.whl (115 kB)\n","\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m115.3/115.3 kB\u001b[0m \u001b[31m11.5 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n","\u001b[?25hDownloading fsspec-2023.10.0-py3-none-any.whl (166 kB)\n","\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m166.4/166.4 kB\u001b[0m \u001b[31m14.4 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n","\u001b[?25hDownloading multiprocess-0.70.15-py311-none-any.whl (135 kB)\n","\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m135.4/135.4 kB\u001b[0m \u001b[31m14.6 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n","\u001b[?25hDownloading pyarrow_hotfix-0.7-py3-none-any.whl (7.9 kB)\n","Installing collected packages: pyarrow-hotfix, fsspec, dill, multiprocess, datasets\n","  Attempting uninstall: fsspec\n","    Found existing installation: fsspec 2025.3.0\n","    Uninstalling fsspec-2025.3.0:\n","      Successfully uninstalled fsspec-2025.3.0\n","  Attempting uninstall: dill\n","    Found existing installation: dill 0.3.8\n","    Uninstalling dill-0.3.8:\n","      Successfully uninstalled dill-0.3.8\n","  Attempting uninstall: multiprocess\n","    Found existing installation: multiprocess 0.70.16\n","    Uninstalling multiprocess-0.70.16:\n","      Successfully uninstalled multiprocess-0.70.16\n","  Attempting uninstall: datasets\n","    Found existing installation: datasets 4.0.0\n","    Uninstalling datasets-4.0.0:\n","      Successfully uninstalled datasets-4.0.0\n","\u001b[31mERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.\n","gcsfs 2025.3.0 requires fsspec==2025.3.0, but you have fsspec 2023.10.0 which is incompatible.\u001b[0m\u001b[31m\n","\u001b[0mSuccessfully installed datasets-2.16.1 dill-0.3.7 fsspec-2023.10.0 multiprocess-0.70.15 pyarrow-hotfix-0.7\n","+ git clone https://github.com/openai/human-eval\n","fatal: destination path 'human-eval' already exists and is not an empty directory.\n","+ pip3 install -e human-eval\n","Obtaining file:///content/drive/MyDrive/CS598JBR-Team-27%20Drive/CS598JBR-Team-0/human-eval\n","  Preparing metadata (setup.py) ... \u001b[?25l\u001b[?25hdone\n","Requirement already satisfied: tqdm in /usr/local/lib/python3.12/dist-packages (from human-eval==1.0) (4.67.1)\n","Collecting fire (from human-eval==1.0)\n","  Downloading fire-0.7.1-py3-none-any.whl.metadata (5.8 kB)\n","Requirement already satisfied: numpy in /usr/local/lib/python3.12/dist-packages (from human-eval==1.0) (2.0.2)\n","Requirement already satisfied: termcolor in /usr/local/lib/python3.12/dist-packages (from fire->human-eval==1.0) (3.1.0)\n","Downloading fire-0.7.1-py3-none-any.whl (115 kB)\n","\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m115.9/115.9 kB\u001b[0m \u001b[31m3.5 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n","\u001b[?25hInstalling collected packages: fire, human-eval\n","  Running setup.py develop for human-eval\n","Successfully installed fire-0.7.1 human-eval-1.0\n","+ sed -i 's/^#\\([[:space:]]*\\)exec(check_program, exec_globals)/\\1exec(check_program, exec_globals)/' human-eval/human_eval/execution.py\n","+ sed -i '/assert len(completion_id) == len(problems), \"Some problems are not attempted.\"/s/^/#/' human-eval/human_eval/evaluation.py\n","+ pip3 install jsonlines\n","Collecting jsonlines\n","  Downloading jsonlines-4.0.0-py3-none-any.whl.metadata (1.6 kB)\n","Requirement already satisfied: attrs>=19.2.0 in /usr/local/lib/python3.12/dist-packages (from jsonlines) (25.3.0)\n","Downloading jsonlines-4.0.0-py3-none-any.whl (8.7 kB)\n","Installing collected packages: jsonlines\n","Successfully installed jsonlines-4.0.0\n","NetIDs ['mae10mns9snh6pranavn6'] with seed 270992647952172920073750662102343813773\n","Downloading readme: 6.52kB [00:00, 20.6MB/s]\n","Downloading data: 100%|██████████| 83.9k/83.9k [00:00<00:00, 450kB/s]\n","Generating test split: 100%|██████████| 164/164 [00:00<00:00, 3641.89 examples/s]\n","Entire Dataset saved to humaneval.jsonl\n","Selected 20 problems saved to selected_humaneval_270992647952172920073750662102343813773.jsonl\n"]}]},{"cell_type":"code","source":["# Set up requirements for model prompting\n","! bash -x MP1/setup_models.sh"],"metadata":{"colab":{"base_uri":"https://localhost:8080/"},"id":"cWg7M56gy_rG","executionInfo":{"status":"ok","timestamp":1758649145568,"user_tz":300,"elapsed":141276,"user":{"displayName":"Maverick Espinosa","userId":"01952844037976753706"}},"outputId":"ae8bd8a8-095f-4dcf-c409-78552f5b7ad3"},"execution_count":null,"outputs":[{"output_type":"stream","name":"stdout","text":["+ pip3 install bitsandbytes==0.47.0\n","Collecting bitsandbytes==0.47.0\n","  Downloading bitsandbytes-0.47.0-py3-none-manylinux_2_24_x86_64.whl.metadata (11 kB)\n","Requirement already satisfied: torch<3,>=2.2 in /usr/local/lib/python3.12/dist-packages (from bitsandbytes==0.47.0) (2.8.0+cu126)\n","Requirement already satisfied: numpy>=1.17 in /usr/local/lib/python3.12/dist-packages (from bitsandbytes==0.47.0) (2.0.2)\n","Requirement already satisfied: filelock in /usr/local/lib/python3.12/dist-packages (from torch<3,>=2.2->bitsandbytes==0.47.0) (3.19.1)\n","Requirement already satisfied: typing-extensions>=4.10.0 in /usr/local/lib/python3.12/dist-packages (from torch<3,>=2.2->bitsandbytes==0.47.0) (4.15.0)\n","Requirement already satisfied: setuptools in /usr/local/lib/python3.12/dist-packages (from torch<3,>=2.2->bitsandbytes==0.47.0) (75.2.0)\n","Requirement already satisfied: sympy>=1.13.3 in /usr/local/lib/python3.12/dist-packages (from torch<3,>=2.2->bitsandbytes==0.47.0) (1.13.3)\n","Requirement already satisfied: networkx in /usr/local/lib/python3.12/dist-packages (from torch<3,>=2.2->bitsandbytes==0.47.0) (3.5)\n","Requirement already satisfied: jinja2 in /usr/local/lib/python3.12/dist-packages (from torch<3,>=2.2->bitsandbytes==0.47.0) (3.1.6)\n","Requirement already satisfied: fsspec in /usr/local/lib/python3.12/dist-packages (from torch<3,>=2.2->bitsandbytes==0.47.0) (2023.10.0)\n","Requirement already satisfied: nvidia-cuda-nvrtc-cu12==12.6.77 in /usr/local/lib/python3.12/dist-packages (from torch<3,>=2.2->bitsandbytes==0.47.0) (12.6.77)\n","Requirement already satisfied: nvidia-cuda-runtime-cu12==12.6.77 in /usr/local/lib/python3.12/dist-packages (from torch<3,>=2.2->bitsandbytes==0.47.0) (12.6.77)\n","Requirement already satisfied: nvidia-cuda-cupti-cu12==12.6.80 in /usr/local/lib/python3.12/dist-packages (from torch<3,>=2.2->bitsandbytes==0.47.0) (12.6.80)\n","Requirement already satisfied: nvidia-cudnn-cu12==9.10.2.21 in /usr/local/lib/python3.12/dist-packages (from torch<3,>=2.2->bitsandbytes==0.47.0) (9.10.2.21)\n","Requirement already satisfied: nvidia-cublas-cu12==12.6.4.1 in /usr/local/lib/python3.12/dist-packages (from torch<3,>=2.2->bitsandbytes==0.47.0) (12.6.4.1)\n","Requirement already satisfied: nvidia-cufft-cu12==11.3.0.4 in /usr/local/lib/python3.12/dist-packages (from torch<3,>=2.2->bitsandbytes==0.47.0) (11.3.0.4)\n","Requirement already satisfied: nvidia-curand-cu12==10.3.7.77 in /usr/local/lib/python3.12/dist-packages (from torch<3,>=2.2->bitsandbytes==0.47.0) (10.3.7.77)\n","Requirement already satisfied: nvidia-cusolver-cu12==11.7.1.2 in /usr/local/lib/python3.12/dist-packages (from torch<3,>=2.2->bitsandbytes==0.47.0) (11.7.1.2)\n","Requirement already satisfied: nvidia-cusparse-cu12==12.5.4.2 in /usr/local/lib/python3.12/dist-packages (from torch<3,>=2.2->bitsandbytes==0.47.0) (12.5.4.2)\n","Requirement already satisfied: nvidia-cusparselt-cu12==0.7.1 in /usr/local/lib/python3.12/dist-packages (from torch<3,>=2.2->bitsandbytes==0.47.0) (0.7.1)\n","Requirement already satisfied: nvidia-nccl-cu12==2.27.3 in /usr/local/lib/python3.12/dist-packages (from torch<3,>=2.2->bitsandbytes==0.47.0) (2.27.3)\n","Requirement already satisfied: nvidia-nvtx-cu12==12.6.77 in /usr/local/lib/python3.12/dist-packages (from torch<3,>=2.2->bitsandbytes==0.47.0) (12.6.77)\n","Requirement already satisfied: nvidia-nvjitlink-cu12==12.6.85 in /usr/local/lib/python3.12/dist-packages (from torch<3,>=2.2->bitsandbytes==0.47.0) (12.6.85)\n","Requirement already satisfied: nvidia-cufile-cu12==1.11.1.6 in /usr/local/lib/python3.12/dist-packages (from torch<3,>=2.2->bitsandbytes==0.47.0) (1.11.1.6)\n","Requirement already satisfied: triton==3.4.0 in /usr/local/lib/python3.12/dist-packages (from torch<3,>=2.2->bitsandbytes==0.47.0) (3.4.0)\n","Requirement already satisfied: mpmath<1.4,>=1.1.0 in /usr/local/lib/python3.12/dist-packages (from sympy>=1.13.3->torch<3,>=2.2->bitsandbytes==0.47.0) (1.3.0)\n","Requirement already satisfied: MarkupSafe>=2.0 in /usr/local/lib/python3.12/dist-packages (from jinja2->torch<3,>=2.2->bitsandbytes==0.47.0) (3.0.2)\n","Downloading bitsandbytes-0.47.0-py3-none-manylinux_2_24_x86_64.whl (61.3 MB)\n","\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m61.3/61.3 MB\u001b[0m \u001b[31m9.0 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n","\u001b[?25hInstalling collected packages: bitsandbytes\n","Successfully installed bitsandbytes-0.47.0\n","+ pip3 install transformers==4.55.4\n","Collecting transformers==4.55.4\n","  Downloading transformers-4.55.4-py3-none-any.whl.metadata (41 kB)\n","\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m42.0/42.0 kB\u001b[0m \u001b[31m1.8 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n","\u001b[?25hRequirement already satisfied: filelock in /usr/local/lib/python3.12/dist-packages (from transformers==4.55.4) (3.19.1)\n","Requirement already satisfied: huggingface-hub<1.0,>=0.34.0 in /usr/local/lib/python3.12/dist-packages (from transformers==4.55.4) (0.35.0)\n","Requirement already satisfied: numpy>=1.17 in /usr/local/lib/python3.12/dist-packages (from transformers==4.55.4) (2.0.2)\n","Requirement already satisfied: packaging>=20.0 in /usr/local/lib/python3.12/dist-packages (from transformers==4.55.4) (25.0)\n","Requirement already satisfied: pyyaml>=5.1 in /usr/local/lib/python3.12/dist-packages (from transformers==4.55.4) (6.0.2)\n","Requirement already satisfied: regex!=2019.12.17 in /usr/local/lib/python3.12/dist-packages (from transformers==4.55.4) (2024.11.6)\n","Requirement already satisfied: requests in /usr/local/lib/python3.12/dist-packages (from transformers==4.55.4) (2.32.4)\n","Collecting tokenizers<0.22,>=0.21 (from transformers==4.55.4)\n","  Downloading tokenizers-0.21.4-cp39-abi3-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (6.7 kB)\n","Requirement already satisfied: safetensors>=0.4.3 in /usr/local/lib/python3.12/dist-packages (from transformers==4.55.4) (0.6.2)\n","Requirement already satisfied: tqdm>=4.27 in /usr/local/lib/python3.12/dist-packages (from transformers==4.55.4) (4.67.1)\n","Requirement already satisfied: fsspec>=2023.5.0 in /usr/local/lib/python3.12/dist-packages (from huggingface-hub<1.0,>=0.34.0->transformers==4.55.4) (2023.10.0)\n","Requirement already satisfied: typing-extensions>=3.7.4.3 in /usr/local/lib/python3.12/dist-packages (from huggingface-hub<1.0,>=0.34.0->transformers==4.55.4) (4.15.0)\n","Requirement already satisfied: hf-xet<2.0.0,>=1.1.3 in /usr/local/lib/python3.12/dist-packages (from huggingface-hub<1.0,>=0.34.0->transformers==4.55.4) (1.1.10)\n","Requirement already satisfied: charset_normalizer<4,>=2 in /usr/local/lib/python3.12/dist-packages (from requests->transformers==4.55.4) (3.4.3)\n","Requirement already satisfied: idna<4,>=2.5 in /usr/local/lib/python3.12/dist-packages (from requests->transformers==4.55.4) (3.10)\n","Requirement already satisfied: urllib3<3,>=1.21.1 in /usr/local/lib/python3.12/dist-packages (from requests->transformers==4.55.4) (2.5.0)\n","Requirement already satisfied: certifi>=2017.4.17 in /usr/local/lib/python3.12/dist-packages (from requests->transformers==4.55.4) (2025.8.3)\n","Downloading transformers-4.55.4-py3-none-any.whl (11.3 MB)\n","\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m11.3/11.3 MB\u001b[0m \u001b[31m118.6 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n","\u001b[?25hDownloading tokenizers-0.21.4-cp39-abi3-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (3.1 MB)\n","\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m3.1/3.1 MB\u001b[0m \u001b[31m125.6 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n","\u001b[?25hInstalling collected packages: tokenizers, transformers\n","  Attempting uninstall: tokenizers\n","    Found existing installation: tokenizers 0.22.0\n","    Uninstalling tokenizers-0.22.0:\n","      Successfully uninstalled tokenizers-0.22.0\n","  Attempting uninstall: transformers\n","    Found existing installation: transformers 4.56.1\n","    Uninstalling transformers-4.56.1:\n","      Successfully uninstalled transformers-4.56.1\n","Successfully installed tokenizers-0.21.4 transformers-4.55.4\n","+ pip3 install peft==0.4.0\n","Collecting peft==0.4.0\n","  Downloading peft-0.4.0-py3-none-any.whl.metadata (21 kB)\n","Requirement already satisfied: numpy>=1.17 in /usr/local/lib/python3.12/dist-packages (from peft==0.4.0) (2.0.2)\n","Requirement already satisfied: packaging>=20.0 in /usr/local/lib/python3.12/dist-packages (from peft==0.4.0) (25.0)\n","Requirement already satisfied: psutil in /usr/local/lib/python3.12/dist-packages (from peft==0.4.0) (5.9.5)\n","Requirement already satisfied: pyyaml in /usr/local/lib/python3.12/dist-packages (from peft==0.4.0) (6.0.2)\n","Requirement already satisfied: torch>=1.13.0 in /usr/local/lib/python3.12/dist-packages (from peft==0.4.0) (2.8.0+cu126)\n","Requirement already satisfied: transformers in /usr/local/lib/python3.12/dist-packages (from peft==0.4.0) (4.55.4)\n","Requirement already satisfied: accelerate in /usr/local/lib/python3.12/dist-packages (from peft==0.4.0) (1.10.1)\n","Requirement already satisfied: safetensors in /usr/local/lib/python3.12/dist-packages (from peft==0.4.0) (0.6.2)\n","Requirement already satisfied: filelock in /usr/local/lib/python3.12/dist-packages (from torch>=1.13.0->peft==0.4.0) (3.19.1)\n","Requirement already satisfied: typing-extensions>=4.10.0 in /usr/local/lib/python3.12/dist-packages (from torch>=1.13.0->peft==0.4.0) (4.15.0)\n","Requirement already satisfied: setuptools in /usr/local/lib/python3.12/dist-packages (from torch>=1.13.0->peft==0.4.0) (75.2.0)\n","Requirement already satisfied: sympy>=1.13.3 in /usr/local/lib/python3.12/dist-packages (from torch>=1.13.0->peft==0.4.0) (1.13.3)\n","Requirement already satisfied: networkx in /usr/local/lib/python3.12/dist-packages (from torch>=1.13.0->peft==0.4.0) (3.5)\n","Requirement already satisfied: jinja2 in /usr/local/lib/python3.12/dist-packages (from torch>=1.13.0->peft==0.4.0) (3.1.6)\n","Requirement already satisfied: fsspec in /usr/local/lib/python3.12/dist-packages (from torch>=1.13.0->peft==0.4.0) (2023.10.0)\n","Requirement already satisfied: nvidia-cuda-nvrtc-cu12==12.6.77 in /usr/local/lib/python3.12/dist-packages (from torch>=1.13.0->peft==0.4.0) (12.6.77)\n","Requirement already satisfied: nvidia-cuda-runtime-cu12==12.6.77 in /usr/local/lib/python3.12/dist-packages (from torch>=1.13.0->peft==0.4.0) (12.6.77)\n","Requirement already satisfied: nvidia-cuda-cupti-cu12==12.6.80 in /usr/local/lib/python3.12/dist-packages (from torch>=1.13.0->peft==0.4.0) (12.6.80)\n","Requirement already satisfied: nvidia-cudnn-cu12==9.10.2.21 in /usr/local/lib/python3.12/dist-packages (from torch>=1.13.0->peft==0.4.0) (9.10.2.21)\n","Requirement already satisfied: nvidia-cublas-cu12==12.6.4.1 in /usr/local/lib/python3.12/dist-packages (from torch>=1.13.0->peft==0.4.0) (12.6.4.1)\n","Requirement already satisfied: nvidia-cufft-cu12==11.3.0.4 in /usr/local/lib/python3.12/dist-packages (from torch>=1.13.0->peft==0.4.0) (11.3.0.4)\n","Requirement already satisfied: nvidia-curand-cu12==10.3.7.77 in /usr/local/lib/python3.12/dist-packages (from torch>=1.13.0->peft==0.4.0) (10.3.7.77)\n","Requirement already satisfied: nvidia-cusolver-cu12==11.7.1.2 in /usr/local/lib/python3.12/dist-packages (from torch>=1.13.0->peft==0.4.0) (11.7.1.2)\n","Requirement already satisfied: nvidia-cusparse-cu12==12.5.4.2 in /usr/local/lib/python3.12/dist-packages (from torch>=1.13.0->peft==0.4.0) (12.5.4.2)\n","Requirement already satisfied: nvidia-cusparselt-cu12==0.7.1 in /usr/local/lib/python3.12/dist-packages (from torch>=1.13.0->peft==0.4.0) (0.7.1)\n","Requirement already satisfied: nvidia-nccl-cu12==2.27.3 in /usr/local/lib/python3.12/dist-packages (from torch>=1.13.0->peft==0.4.0) (2.27.3)\n","Requirement already satisfied: nvidia-nvtx-cu12==12.6.77 in /usr/local/lib/python3.12/dist-packages (from torch>=1.13.0->peft==0.4.0) (12.6.77)\n","Requirement already satisfied: nvidia-nvjitlink-cu12==12.6.85 in /usr/local/lib/python3.12/dist-packages (from torch>=1.13.0->peft==0.4.0) (12.6.85)\n","Requirement already satisfied: nvidia-cufile-cu12==1.11.1.6 in /usr/local/lib/python3.12/dist-packages (from torch>=1.13.0->peft==0.4.0) (1.11.1.6)\n","Requirement already satisfied: triton==3.4.0 in /usr/local/lib/python3.12/dist-packages (from torch>=1.13.0->peft==0.4.0) (3.4.0)\n","Requirement already satisfied: huggingface_hub>=0.21.0 in /usr/local/lib/python3.12/dist-packages (from accelerate->peft==0.4.0) (0.35.0)\n","Requirement already satisfied: regex!=2019.12.17 in /usr/local/lib/python3.12/dist-packages (from transformers->peft==0.4.0) (2024.11.6)\n","Requirement already satisfied: requests in /usr/local/lib/python3.12/dist-packages (from transformers->peft==0.4.0) (2.32.4)\n","Requirement already satisfied: tokenizers<0.22,>=0.21 in /usr/local/lib/python3.12/dist-packages (from transformers->peft==0.4.0) (0.21.4)\n","Requirement already satisfied: tqdm>=4.27 in /usr/local/lib/python3.12/dist-packages (from transformers->peft==0.4.0) (4.67.1)\n","Requirement already satisfied: hf-xet<2.0.0,>=1.1.3 in /usr/local/lib/python3.12/dist-packages (from huggingface_hub>=0.21.0->accelerate->peft==0.4.0) (1.1.10)\n","Requirement already satisfied: mpmath<1.4,>=1.1.0 in /usr/local/lib/python3.12/dist-packages (from sympy>=1.13.3->torch>=1.13.0->peft==0.4.0) (1.3.0)\n","Requirement already satisfied: MarkupSafe>=2.0 in /usr/local/lib/python3.12/dist-packages (from jinja2->torch>=1.13.0->peft==0.4.0) (3.0.2)\n","Requirement already satisfied: charset_normalizer<4,>=2 in /usr/local/lib/python3.12/dist-packages (from requests->transformers->peft==0.4.0) (3.4.3)\n","Requirement already satisfied: idna<4,>=2.5 in /usr/local/lib/python3.12/dist-packages (from requests->transformers->peft==0.4.0) (3.10)\n","Requirement already satisfied: urllib3<3,>=1.21.1 in /usr/local/lib/python3.12/dist-packages (from requests->transformers->peft==0.4.0) (2.5.0)\n","Requirement already satisfied: certifi>=2017.4.17 in /usr/local/lib/python3.12/dist-packages (from requests->transformers->peft==0.4.0) (2025.8.3)\n","Downloading peft-0.4.0-py3-none-any.whl (72 kB)\n","\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m72.9/72.9 kB\u001b[0m \u001b[31m2.9 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n","\u001b[?25hInstalling collected packages: peft\n","  Attempting uninstall: peft\n","    Found existing installation: peft 0.17.1\n","    Uninstalling peft-0.17.1:\n","      Successfully uninstalled peft-0.17.1\n","Successfully installed peft-0.4.0\n","+ pip3 install accelerate==1.10.1\n","Requirement already satisfied: accelerate==1.10.1 in /usr/local/lib/python3.12/dist-packages (1.10.1)\n","Requirement already satisfied: numpy<3.0.0,>=1.17 in /usr/local/lib/python3.12/dist-packages (from accelerate==1.10.1) (2.0.2)\n","Requirement already satisfied: packaging>=20.0 in /usr/local/lib/python3.12/dist-packages (from accelerate==1.10.1) (25.0)\n","Requirement already satisfied: psutil in /usr/local/lib/python3.12/dist-packages (from accelerate==1.10.1) (5.9.5)\n","Requirement already satisfied: pyyaml in /usr/local/lib/python3.12/dist-packages (from accelerate==1.10.1) (6.0.2)\n","Requirement already satisfied: torch>=2.0.0 in /usr/local/lib/python3.12/dist-packages (from accelerate==1.10.1) (2.8.0+cu126)\n","Requirement already satisfied: huggingface_hub>=0.21.0 in /usr/local/lib/python3.12/dist-packages (from accelerate==1.10.1) (0.35.0)\n","Requirement already satisfied: safetensors>=0.4.3 in /usr/local/lib/python3.12/dist-packages (from accelerate==1.10.1) (0.6.2)\n","Requirement already satisfied: filelock in /usr/local/lib/python3.12/dist-packages (from huggingface_hub>=0.21.0->accelerate==1.10.1) (3.19.1)\n","Requirement already satisfied: fsspec>=2023.5.0 in /usr/local/lib/python3.12/dist-packages (from huggingface_hub>=0.21.0->accelerate==1.10.1) (2023.10.0)\n","Requirement already satisfied: requests in /usr/local/lib/python3.12/dist-packages (from huggingface_hub>=0.21.0->accelerate==1.10.1) (2.32.4)\n","Requirement already satisfied: tqdm>=4.42.1 in /usr/local/lib/python3.12/dist-packages (from huggingface_hub>=0.21.0->accelerate==1.10.1) (4.67.1)\n","Requirement already satisfied: typing-extensions>=3.7.4.3 in /usr/local/lib/python3.12/dist-packages (from huggingface_hub>=0.21.0->accelerate==1.10.1) (4.15.0)\n","Requirement already satisfied: hf-xet<2.0.0,>=1.1.3 in /usr/local/lib/python3.12/dist-packages (from huggingface_hub>=0.21.0->accelerate==1.10.1) (1.1.10)\n","Requirement already satisfied: setuptools in /usr/local/lib/python3.12/dist-packages (from torch>=2.0.0->accelerate==1.10.1) (75.2.0)\n","Requirement already satisfied: sympy>=1.13.3 in /usr/local/lib/python3.12/dist-packages (from torch>=2.0.0->accelerate==1.10.1) (1.13.3)\n","Requirement already satisfied: networkx in /usr/local/lib/python3.12/dist-packages (from torch>=2.0.0->accelerate==1.10.1) (3.5)\n","Requirement already satisfied: jinja2 in /usr/local/lib/python3.12/dist-packages (from torch>=2.0.0->accelerate==1.10.1) (3.1.6)\n","Requirement already satisfied: nvidia-cuda-nvrtc-cu12==12.6.77 in /usr/local/lib/python3.12/dist-packages (from torch>=2.0.0->accelerate==1.10.1) (12.6.77)\n","Requirement already satisfied: nvidia-cuda-runtime-cu12==12.6.77 in /usr/local/lib/python3.12/dist-packages (from torch>=2.0.0->accelerate==1.10.1) (12.6.77)\n","Requirement already satisfied: nvidia-cuda-cupti-cu12==12.6.80 in /usr/local/lib/python3.12/dist-packages (from torch>=2.0.0->accelerate==1.10.1) (12.6.80)\n","Requirement already satisfied: nvidia-cudnn-cu12==9.10.2.21 in /usr/local/lib/python3.12/dist-packages (from torch>=2.0.0->accelerate==1.10.1) (9.10.2.21)\n","Requirement already satisfied: nvidia-cublas-cu12==12.6.4.1 in /usr/local/lib/python3.12/dist-packages (from torch>=2.0.0->accelerate==1.10.1) (12.6.4.1)\n","Requirement already satisfied: nvidia-cufft-cu12==11.3.0.4 in /usr/local/lib/python3.12/dist-packages (from torch>=2.0.0->accelerate==1.10.1) (11.3.0.4)\n","Requirement already satisfied: nvidia-curand-cu12==10.3.7.77 in /usr/local/lib/python3.12/dist-packages (from torch>=2.0.0->accelerate==1.10.1) (10.3.7.77)\n","Requirement already satisfied: nvidia-cusolver-cu12==11.7.1.2 in /usr/local/lib/python3.12/dist-packages (from torch>=2.0.0->accelerate==1.10.1) (11.7.1.2)\n","Requirement already satisfied: nvidia-cusparse-cu12==12.5.4.2 in /usr/local/lib/python3.12/dist-packages (from torch>=2.0.0->accelerate==1.10.1) (12.5.4.2)\n","Requirement already satisfied: nvidia-cusparselt-cu12==0.7.1 in /usr/local/lib/python3.12/dist-packages (from torch>=2.0.0->accelerate==1.10.1) (0.7.1)\n","Requirement already satisfied: nvidia-nccl-cu12==2.27.3 in /usr/local/lib/python3.12/dist-packages (from torch>=2.0.0->accelerate==1.10.1) (2.27.3)\n","Requirement already satisfied: nvidia-nvtx-cu12==12.6.77 in /usr/local/lib/python3.12/dist-packages (from torch>=2.0.0->accelerate==1.10.1) (12.6.77)\n","Requirement already satisfied: nvidia-nvjitlink-cu12==12.6.85 in /usr/local/lib/python3.12/dist-packages (from torch>=2.0.0->accelerate==1.10.1) (12.6.85)\n","Requirement already satisfied: nvidia-cufile-cu12==1.11.1.6 in /usr/local/lib/python3.12/dist-packages (from torch>=2.0.0->accelerate==1.10.1) (1.11.1.6)\n","Requirement already satisfied: triton==3.4.0 in /usr/local/lib/python3.12/dist-packages (from torch>=2.0.0->accelerate==1.10.1) (3.4.0)\n","Requirement already satisfied: mpmath<1.4,>=1.1.0 in /usr/local/lib/python3.12/dist-packages (from sympy>=1.13.3->torch>=2.0.0->accelerate==1.10.1) (1.3.0)\n","Requirement already satisfied: MarkupSafe>=2.0 in /usr/local/lib/python3.12/dist-packages (from jinja2->torch>=2.0.0->accelerate==1.10.1) (3.0.2)\n","Requirement already satisfied: charset_normalizer<4,>=2 in /usr/local/lib/python3.12/dist-packages (from requests->huggingface_hub>=0.21.0->accelerate==1.10.1) (3.4.3)\n","Requirement already satisfied: idna<4,>=2.5 in /usr/local/lib/python3.12/dist-packages (from requests->huggingface_hub>=0.21.0->accelerate==1.10.1) (3.10)\n","Requirement already satisfied: urllib3<3,>=1.21.1 in /usr/local/lib/python3.12/dist-packages (from requests->huggingface_hub>=0.21.0->accelerate==1.10.1) (2.5.0)\n","Requirement already satisfied: certifi>=2017.4.17 in /usr/local/lib/python3.12/dist-packages (from requests->huggingface_hub>=0.21.0->accelerate==1.10.1) (2025.8.3)\n","+ pip3 install einops==0.6.1\n","Collecting einops==0.6.1\n","  Downloading einops-0.6.1-py3-none-any.whl.metadata (12 kB)\n","Downloading einops-0.6.1-py3-none-any.whl (42 kB)\n","\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m42.2/42.2 kB\u001b[0m \u001b[31m2.0 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n","\u001b[?25hInstalling collected packages: einops\n","  Attempting uninstall: einops\n","    Found existing installation: einops 0.8.1\n","    Uninstalling einops-0.8.1:\n","      Successfully uninstalled einops-0.8.1\n","Successfully installed einops-0.6.1\n","+ pip3 install evaluate==0.4.0\n","Collecting evaluate==0.4.0\n","  Downloading evaluate-0.4.0-py3-none-any.whl.metadata (9.4 kB)\n","Requirement already satisfied: datasets>=2.0.0 in /usr/local/lib/python3.12/dist-packages (from evaluate==0.4.0) (2.16.1)\n","Requirement already satisfied: numpy>=1.17 in /usr/local/lib/python3.12/dist-packages (from evaluate==0.4.0) (2.0.2)\n","Requirement already satisfied: dill in /usr/local/lib/python3.12/dist-packages (from evaluate==0.4.0) (0.3.7)\n","Requirement already satisfied: pandas in /usr/local/lib/python3.12/dist-packages (from evaluate==0.4.0) (2.2.2)\n","Requirement already satisfied: requests>=2.19.0 in /usr/local/lib/python3.12/dist-packages (from evaluate==0.4.0) (2.32.4)\n","Requirement already satisfied: tqdm>=4.62.1 in /usr/local/lib/python3.12/dist-packages (from evaluate==0.4.0) (4.67.1)\n","Requirement already satisfied: xxhash in /usr/local/lib/python3.12/dist-packages (from evaluate==0.4.0) (3.5.0)\n","Requirement already satisfied: multiprocess in /usr/local/lib/python3.12/dist-packages (from evaluate==0.4.0) (0.70.15)\n","Requirement already satisfied: fsspec>=2021.05.0 in /usr/local/lib/python3.12/dist-packages (from fsspec[http]>=2021.05.0->evaluate==0.4.0) (2023.10.0)\n","Requirement already satisfied: huggingface-hub>=0.7.0 in /usr/local/lib/python3.12/dist-packages (from evaluate==0.4.0) (0.35.0)\n","Requirement already satisfied: packaging in /usr/local/lib/python3.12/dist-packages (from evaluate==0.4.0) (25.0)\n","Collecting responses<0.19 (from evaluate==0.4.0)\n","  Downloading responses-0.18.0-py3-none-any.whl.metadata (29 kB)\n","Requirement already satisfied: filelock in /usr/local/lib/python3.12/dist-packages (from datasets>=2.0.0->evaluate==0.4.0) (3.19.1)\n","Requirement already satisfied: pyarrow>=8.0.0 in /usr/local/lib/python3.12/dist-packages (from datasets>=2.0.0->evaluate==0.4.0) (18.1.0)\n","Requirement already satisfied: pyarrow-hotfix in /usr/local/lib/python3.12/dist-packages (from datasets>=2.0.0->evaluate==0.4.0) (0.7)\n","Requirement already satisfied: aiohttp in /usr/local/lib/python3.12/dist-packages (from datasets>=2.0.0->evaluate==0.4.0) (3.12.15)\n","Requirement already satisfied: pyyaml>=5.1 in /usr/local/lib/python3.12/dist-packages (from datasets>=2.0.0->evaluate==0.4.0) (6.0.2)\n","Requirement already satisfied: typing-extensions>=3.7.4.3 in /usr/local/lib/python3.12/dist-packages (from huggingface-hub>=0.7.0->evaluate==0.4.0) (4.15.0)\n","Requirement already satisfied: hf-xet<2.0.0,>=1.1.3 in /usr/local/lib/python3.12/dist-packages (from huggingface-hub>=0.7.0->evaluate==0.4.0) (1.1.10)\n","Requirement already satisfied: charset_normalizer<4,>=2 in /usr/local/lib/python3.12/dist-packages (from requests>=2.19.0->evaluate==0.4.0) (3.4.3)\n","Requirement already satisfied: idna<4,>=2.5 in /usr/local/lib/python3.12/dist-packages (from requests>=2.19.0->evaluate==0.4.0) (3.10)\n","Requirement already satisfied: urllib3<3,>=1.21.1 in /usr/local/lib/python3.12/dist-packages (from requests>=2.19.0->evaluate==0.4.0) (2.5.0)\n","Requirement already satisfied: certifi>=2017.4.17 in /usr/local/lib/python3.12/dist-packages (from requests>=2.19.0->evaluate==0.4.0) (2025.8.3)\n","Requirement already satisfied: python-dateutil>=2.8.2 in /usr/local/lib/python3.12/dist-packages (from pandas->evaluate==0.4.0) (2.9.0.post0)\n","Requirement already satisfied: pytz>=2020.1 in /usr/local/lib/python3.12/dist-packages (from pandas->evaluate==0.4.0) (2025.2)\n","Requirement already satisfied: tzdata>=2022.7 in /usr/local/lib/python3.12/dist-packages (from pandas->evaluate==0.4.0) (2025.2)\n","Requirement already satisfied: aiohappyeyeballs>=2.5.0 in /usr/local/lib/python3.12/dist-packages (from aiohttp->datasets>=2.0.0->evaluate==0.4.0) (2.6.1)\n","Requirement already satisfied: aiosignal>=1.4.0 in /usr/local/lib/python3.12/dist-packages (from aiohttp->datasets>=2.0.0->evaluate==0.4.0) (1.4.0)\n","Requirement already satisfied: attrs>=17.3.0 in /usr/local/lib/python3.12/dist-packages (from aiohttp->datasets>=2.0.0->evaluate==0.4.0) (25.3.0)\n","Requirement already satisfied: frozenlist>=1.1.1 in /usr/local/lib/python3.12/dist-packages (from aiohttp->datasets>=2.0.0->evaluate==0.4.0) (1.7.0)\n","Requirement already satisfied: multidict<7.0,>=4.5 in /usr/local/lib/python3.12/dist-packages (from aiohttp->datasets>=2.0.0->evaluate==0.4.0) (6.6.4)\n","Requirement already satisfied: propcache>=0.2.0 in /usr/local/lib/python3.12/dist-packages (from aiohttp->datasets>=2.0.0->evaluate==0.4.0) (0.3.2)\n","Requirement already satisfied: yarl<2.0,>=1.17.0 in /usr/local/lib/python3.12/dist-packages (from aiohttp->datasets>=2.0.0->evaluate==0.4.0) (1.20.1)\n","Requirement already satisfied: six>=1.5 in /usr/local/lib/python3.12/dist-packages (from python-dateutil>=2.8.2->pandas->evaluate==0.4.0) (1.17.0)\n","Downloading evaluate-0.4.0-py3-none-any.whl (81 kB)\n","\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m81.4/81.4 kB\u001b[0m \u001b[31m3.7 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n","\u001b[?25hDownloading responses-0.18.0-py3-none-any.whl (38 kB)\n","Installing collected packages: responses, evaluate\n","Successfully installed evaluate-0.4.0 responses-0.18.0\n","+ pip3 install sentencepiece==0.1.99\n","Collecting sentencepiece==0.1.99\n","  Downloading sentencepiece-0.1.99.tar.gz (2.6 MB)\n","\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m2.6/2.6 MB\u001b[0m \u001b[31m30.7 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n","\u001b[?25h  Preparing metadata (setup.py) ... \u001b[?25l\u001b[?25hdone\n","Building wheels for collected packages: sentencepiece\n","  Building wheel for sentencepiece (setup.py) ... \u001b[?25l\u001b[?25hdone\n","  Created wheel for sentencepiece: filename=sentencepiece-0.1.99-cp312-cp312-linux_x86_64.whl size=1266901 sha256=bd06bbd950559a8cc6336ff123c1368e2d2903b28ff80ee3dfb84f50664bb29f\n","  Stored in directory: /root/.cache/pip/wheels/e0/8c/e0/65e33b1f4b8462dfc537a0cac02e5c03e1207564c300e4bde5\n","Successfully built sentencepiece\n","Installing collected packages: sentencepiece\n","  Attempting uninstall: sentencepiece\n","    Found existing installation: sentencepiece 0.2.1\n","    Uninstalling sentencepiece-0.2.1:\n","      Successfully uninstalled sentencepiece-0.2.1\n","Successfully installed sentencepiece-0.1.99\n"]}]},{"cell_type":"code","source":["!pwd\n","seed = \"270992647952172920073750662102343813773\"\n","# TODO: Replace the file path of selected_humaneval_[seed].jsonl generated in previous step\n","input_dataset = \"selected_humaneval_\" + seed + \".jsonl\"\n","print(input_dataset)"],"metadata":{"colab":{"base_uri":"https://localhost:8080/"},"id":"fXUMm9evqkzE","executionInfo":{"status":"ok","timestamp":1758649145682,"user_tz":300,"elapsed":112,"user":{"displayName":"Maverick Espinosa","userId":"01952844037976753706"}},"outputId":"367d0af5-bc61-4c7e-b26d-acd21c5fd9f8"},"execution_count":null,"outputs":[{"output_type":"stream","name":"stdout","text":["/content/drive/MyDrive/CS598JBR-Team-27 Drive/CS598JBR-Team-0\n","selected_humaneval_270992647952172920073750662102343813773.jsonl\n"]}]},{"cell_type":"code","source":["base_with_quantization = \"base_prompt_\" + seed + \".jsonl\"\n","instruct_with_quantization = \"instruct_prompt_\" + seed + \".jsonl\"\n","base_with_quantization_processed = \"base_prompt_processed_\" + seed + \".jsonl\"\n","instruct_with_quantization_processed = \"instruct_prompt_processed_\" + seed + \".jsonl\"\n","\n","# Prompt the models\n","! python3 MP1/model_prompting.py {input_dataset} \"deepseek-ai/deepseek-coder-6.7b-base\" {base_with_quantization} {base_with_quantization_processed} \"True\" |& tee base_prompt.log\n","! python3 MP1/model_prompting.py {input_dataset} \"deepseek-ai/deepseek-coder-6.7b-instruct\" {instruct_with_quantization} {instruct_with_quantization_processed} \"True\" |& tee instruct_prompt.log"],"metadata":{"colab":{"base_uri":"https://localhost:8080/"},"id":"yLgdqUJTwESV","outputId":"29794777-4539-490b-f83b-3f0b1f4eba9e"},"execution_count":null,"outputs":[{"output_type":"stream","name":"stdout","text":["2025-09-23 18:05:57.101034: E external/local_xla/xla/stream_executor/cuda/cuda_fft.cc:467] Unable to register cuFFT factory: Attempting to register factory for plugin cuFFT when one has already been registered\n","WARNING: All log messages before absl::InitializeLog() is called are written to STDERR\n","E0000 00:00:1758650757.396322   11846 cuda_dnn.cc:8579] Unable to register cuDNN factory: Attempting to register factory for plugin cuDNN when one has already been registered\n","E0000 00:00:1758650757.471908   11846 cuda_blas.cc:1407] Unable to register cuBLAS factory: Attempting to register factory for plugin cuBLAS when one has already been registered\n","W0000 00:00:1758650758.051909   11846 computation_placer.cc:177] computation placer already registered. Please check linkage and avoid linking the same target more than once.\n","W0000 00:00:1758650758.051944   11846 computation_placer.cc:177] computation placer already registered. Please check linkage and avoid linking the same target more than once.\n","W0000 00:00:1758650758.051947   11846 computation_placer.cc:177] computation placer already registered. Please check linkage and avoid linking the same target more than once.\n","W0000 00:00:1758650758.051950   11846 computation_placer.cc:177] computation placer already registered. Please check linkage and avoid linking the same target more than once.\n","2025-09-23 18:05:58.107665: I tensorflow/core/platform/cpu_feature_guard.cc:210] This TensorFlow binary is optimized to use available CPU instructions in performance-critical operations.\n","To enable the following instructions: AVX2 FMA, in other operations, rebuild TensorFlow with the appropriate compiler flags.\n","Working with deepseek-ai/deepseek-coder-6.7b-base, quantization=True...\n","Downloading model (if not cached)...\n","Loading checkpoint shards:   0%|          | 0/2 [00:00<?, ?it/s]"]}]},{"cell_type":"code","source":["\n","\n","# evaluate the results to get pass@k\n","! evaluate_functional_correctness {base_with_quantization} |& tee base_evaluate.log\n","! evaluate_functional_correctness {instruct_with_quantization} |& tee instruct_evaluate.log\n","! evaluate_functional_correctness {base_with_quantization_processed} |& tee base_evaluate_processed.log\n","! evaluate_functional_correctness {instruct_with_quantization_processed} |& tee instruct_evaluate_processed.log\n","\n","# %cd ..\n","# git push all nessacery files (e.g., *jsonl, *log) to your GitHub repository"],"metadata":{"id":"E54XUtImtwg6","colab":{"base_uri":"https://localhost:8080/"},"executionInfo":{"status":"ok","timestamp":1758650203930,"user_tz":300,"elapsed":1436,"user":{"displayName":"Maverick Espinosa","userId":"01952844037976753706"}},"outputId":"fd905f27-1235-449b-8a96-7214dea205a9"},"execution_count":null,"outputs":[{"output_type":"stream","name":"stdout","text":["Reading samples...\n","\r0it [00:00, ?it/s]\r0it [00:00, ?it/s]\n","Traceback (most recent call last):\n","  File \"/usr/local/bin/evaluate_functional_correctness\", line 33, in <module>\n","    sys.exit(load_entry_point('human-eval', 'console_scripts', 'evaluate_functional_correctness')())\n","             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n","  File \"/usr/local/bin/evaluate_functional_correctness\", line 25, in importlib_load_entry_point\n","    return next(matches).load()\n","           ^^^^^^^^^^^^^^^^^^^^\n","  File \"/usr/lib/python3.12/importlib/metadata/__init__.py\", line 205, in load\n","    module = import_module(match.group('module'))\n","             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n","  File \"/usr/lib/python3.12/importlib/__init__.py\", line 90, in import_module\n","    return _bootstrap._gcd_import(name[level:], package, level)\n","           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n","  File \"<frozen importlib._bootstrap>\", line 1387, in _gcd_import\n","  File \"<frozen importlib._bootstrap>\", line 1360, in _find_and_load\n","  File \"<frozen importlib._bootstrap>\", line 1331, in _find_and_load_unlocked\n","  File \"<frozen importlib._bootstrap>\", line 935, in _load_unlocked\n","  File \"<frozen importlib._bootstrap_external>\", line 999, in exec_module\n","  File \"<frozen importlib._bootstrap>\", line 488, in _call_with_frames_removed\n","  File \"/content/drive/MyDrive/CS598JBR-Team-27 Drive/CS598JBR-Team-0/human-eval/human_eval/evaluate_functional_correctness.py\", line 28, in <module>\n","    sys.exit(main())\n","             ^^^^^^\n","  File \"/content/drive/MyDrive/CS598JBR-Team-27 Drive/CS598JBR-Team-0/human-eval/human_eval/evaluate_functional_correctness.py\", line 25, in main\n","    fire.Fire(entry_point)\n","  File \"/usr/local/lib/python3.12/dist-packages/fire/core.py\", line 135, in Fire\n","    component_trace = _Fire(component, args, parsed_flag_args, context, name)\n","                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n","  File \"/usr/local/lib/python3.12/dist-packages/fire/core.py\", line 468, in _Fire\n","    component, remaining_args = _CallAndUpdateTrace(\n","                                ^^^^^^^^^^^^^^^^^^^^\n","  File \"/usr/local/lib/python3.12/dist-packages/fire/core.py\", line 684, in _CallAndUpdateTrace\n","    component = fn(*varargs, **kwargs)\n","                ^^^^^^^^^^^^^^^^^^^^^^\n","  File \"/content/drive/MyDrive/CS598JBR-Team-27 Drive/CS598JBR-Team-0/human-eval/human_eval/evaluate_functional_correctness.py\", line 20, in entry_point\n","    results = evaluate_functional_correctness(sample_file, k, n_workers, timeout, problem_file)\n","              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n","  File \"/content/drive/MyDrive/CS598JBR-Team-27 Drive/CS598JBR-Team-0/human-eval/human_eval/evaluation.py\", line 62, in evaluate_functional_correctness\n","    for sample in tqdm.tqdm(stream_jsonl(sample_file)):\n","                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n","  File \"/usr/local/lib/python3.12/dist-packages/tqdm/std.py\", line 1181, in __iter__\n","    for obj in iterable:\n","               ^^^^^^^^\n","  File \"/content/drive/MyDrive/CS598JBR-Team-27 Drive/CS598JBR-Team-0/human-eval/human_eval/data.py\", line 26, in stream_jsonl\n","    with open(filename, \"r\") as fp:\n","         ^^^^^^^^^^^^^^^^^^^\n","FileNotFoundError: [Errno 2] No such file or directory: 'base_prompt_270992647952172920073750662102343813773.jsonl'\n","Reading samples...\n","0it [00:00, ?it/s]\n","Traceback (most recent call last):\n","  File \"/usr/local/bin/evaluate_functional_correctness\", line 33, in <module>\n","    sys.exit(load_entry_point('human-eval', 'console_scripts', 'evaluate_functional_correctness')())\n","             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n","  File \"/usr/local/bin/evaluate_functional_correctness\", line 25, in importlib_load_entry_point\n","    return next(matches).load()\n","           ^^^^^^^^^^^^^^^^^^^^\n","  File \"/usr/lib/python3.12/importlib/metadata/__init__.py\", line 205, in load\n","    module = import_module(match.group('module'))\n","             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n","  File \"/usr/lib/python3.12/importlib/__init__.py\", line 90, in import_module\n","    return _bootstrap._gcd_import(name[level:], package, level)\n","           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n","  File \"<frozen importlib._bootstrap>\", line 1387, in _gcd_import\n","  File \"<frozen importlib._bootstrap>\", line 1360, in _find_and_load\n","  File \"<frozen importlib._bootstrap>\", line 1331, in _find_and_load_unlocked\n","  File \"<frozen importlib._bootstrap>\", line 935, in _load_unlocked\n","  File \"<frozen importlib._bootstrap_external>\", line 999, in exec_module\n","  File \"<frozen importlib._bootstrap>\", line 488, in _call_with_frames_removed\n","  File \"/content/drive/MyDrive/CS598JBR-Team-27 Drive/CS598JBR-Team-0/human-eval/human_eval/evaluate_functional_correctness.py\", line 28, in <module>\n","    sys.exit(main())\n","             ^^^^^^\n","  File \"/content/drive/MyDrive/CS598JBR-Team-27 Drive/CS598JBR-Team-0/human-eval/human_eval/evaluate_functional_correctness.py\", line 25, in main\n","    fire.Fire(entry_point)\n","  File \"/usr/local/lib/python3.12/dist-packages/fire/core.py\", line 135, in Fire\n","    component_trace = _Fire(component, args, parsed_flag_args, context, name)\n","                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n","  File \"/usr/local/lib/python3.12/dist-packages/fire/core.py\", line 468, in _Fire\n","    component, remaining_args = _CallAndUpdateTrace(\n","                                ^^^^^^^^^^^^^^^^^^^^\n","  File \"/usr/local/lib/python3.12/dist-packages/fire/core.py\", line 684, in _CallAndUpdateTrace\n","    component = fn(*varargs, **kwargs)\n","                ^^^^^^^^^^^^^^^^^^^^^^\n","  File \"/content/drive/MyDrive/CS598JBR-Team-27 Drive/CS598JBR-Team-0/human-eval/human_eval/evaluate_functional_correctness.py\", line 20, in entry_point\n","    results = evaluate_functional_correctness(sample_file, k, n_workers, timeout, problem_file)\n","              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n","  File \"/content/drive/MyDrive/CS598JBR-Team-27 Drive/CS598JBR-Team-0/human-eval/human_eval/evaluation.py\", line 62, in evaluate_functional_correctness\n","    for sample in tqdm.tqdm(stream_jsonl(sample_file)):\n","                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n","  File \"/usr/local/lib/python3.12/dist-packages/tqdm/std.py\", line 1181, in __iter__\n","    for obj in iterable:\n","               ^^^^^^^^\n","  File \"/content/drive/MyDrive/CS598JBR-Team-27 Drive/CS598JBR-Team-0/human-eval/human_eval/data.py\", line 26, in stream_jsonl\n","    with open(filename, \"r\") as fp:\n","         ^^^^^^^^^^^^^^^^^^^\n","FileNotFoundError: [Errno 2] No such file or directory: 'instruct_prompt_270992647952172920073750662102343813773.jsonl'\n","Reading samples...\n","0it [00:00, ?it/s]\n","Traceback (most recent call last):\n","  File \"/usr/local/bin/evaluate_functional_correctness\", line 33, in <module>\n","    sys.exit(load_entry_point('human-eval', 'console_scripts', 'evaluate_functional_correctness')())\n","             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n","  File \"/usr/local/bin/evaluate_functional_correctness\", line 25, in importlib_load_entry_point\n","    return next(matches).load()\n","           ^^^^^^^^^^^^^^^^^^^^\n","  File \"/usr/lib/python3.12/importlib/metadata/__init__.py\", line 205, in load\n","    module = import_module(match.group('module'))\n","             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n","  File \"/usr/lib/python3.12/importlib/__init__.py\", line 90, in import_module\n","    return _bootstrap._gcd_import(name[level:], package, level)\n","           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n","  File \"<frozen importlib._bootstrap>\", line 1387, in _gcd_import\n","  File \"<frozen importlib._bootstrap>\", line 1360, in _find_and_load\n","  File \"<frozen importlib._bootstrap>\", line 1331, in _find_and_load_unlocked\n","  File \"<frozen importlib._bootstrap>\", line 935, in _load_unlocked\n","  File \"<frozen importlib._bootstrap_external>\", line 999, in exec_module\n","  File \"<frozen importlib._bootstrap>\", line 488, in _call_with_frames_removed\n","  File \"/content/drive/MyDrive/CS598JBR-Team-27 Drive/CS598JBR-Team-0/human-eval/human_eval/evaluate_functional_correctness.py\", line 28, in <module>\n","    sys.exit(main())\n","             ^^^^^^\n","  File \"/content/drive/MyDrive/CS598JBR-Team-27 Drive/CS598JBR-Team-0/human-eval/human_eval/evaluate_functional_correctness.py\", line 25, in main\n","    fire.Fire(entry_point)\n","  File \"/usr/local/lib/python3.12/dist-packages/fire/core.py\", line 135, in Fire\n","    component_trace = _Fire(component, args, parsed_flag_args, context, name)\n","                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n","  File \"/usr/local/lib/python3.12/dist-packages/fire/core.py\", line 468, in _Fire\n","    component, remaining_args = _CallAndUpdateTrace(\n","                                ^^^^^^^^^^^^^^^^^^^^\n","  File \"/usr/local/lib/python3.12/dist-packages/fire/core.py\", line 684, in _CallAndUpdateTrace\n","    component = fn(*varargs, **kwargs)\n","                ^^^^^^^^^^^^^^^^^^^^^^\n","  File \"/content/drive/MyDrive/CS598JBR-Team-27 Drive/CS598JBR-Team-0/human-eval/human_eval/evaluate_functional_correctness.py\", line 20, in entry_point\n","    results = evaluate_functional_correctness(sample_file, k, n_workers, timeout, problem_file)\n","              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n","  File \"/content/drive/MyDrive/CS598JBR-Team-27 Drive/CS598JBR-Team-0/human-eval/human_eval/evaluation.py\", line 62, in evaluate_functional_correctness\n","    for sample in tqdm.tqdm(stream_jsonl(sample_file)):\n","                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n","  File \"/usr/local/lib/python3.12/dist-packages/tqdm/std.py\", line 1181, in __iter__\n","    for obj in iterable:\n","               ^^^^^^^^\n","  File \"/content/drive/MyDrive/CS598JBR-Team-27 Drive/CS598JBR-Team-0/human-eval/human_eval/data.py\", line 26, in stream_jsonl\n","    with open(filename, \"r\") as fp:\n","         ^^^^^^^^^^^^^^^^^^^\n","FileNotFoundError: [Errno 2] No such file or directory: 'base_prompt_processed_270992647952172920073750662102343813773.jsonl'\n","Reading samples...\n","0it [00:00, ?it/s]\n","Traceback (most recent call last):\n","  File \"/usr/local/bin/evaluate_functional_correctness\", line 33, in <module>\n","    sys.exit(load_entry_point('human-eval', 'console_scripts', 'evaluate_functional_correctness')())\n","             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n","  File \"/usr/local/bin/evaluate_functional_correctness\", line 25, in importlib_load_entry_point\n","    return next(matches).load()\n","           ^^^^^^^^^^^^^^^^^^^^\n","  File \"/usr/lib/python3.12/importlib/metadata/__init__.py\", line 205, in load\n","    module = import_module(match.group('module'))\n","             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n","  File \"/usr/lib/python3.12/importlib/__init__.py\", line 90, in import_module\n","    return _bootstrap._gcd_import(name[level:], package, level)\n","           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n","  File \"<frozen importlib._bootstrap>\", line 1387, in _gcd_import\n","  File \"<frozen importlib._bootstrap>\", line 1360, in _find_and_load\n","  File \"<frozen importlib._bootstrap>\", line 1331, in _find_and_load_unlocked\n","  File \"<frozen importlib._bootstrap>\", line 935, in _load_unlocked\n","  File \"<frozen importlib._bootstrap_external>\", line 999, in exec_module\n","  File \"<frozen importlib._bootstrap>\", line 488, in _call_with_frames_removed\n","  File \"/content/drive/MyDrive/CS598JBR-Team-27 Drive/CS598JBR-Team-0/human-eval/human_eval/evaluate_functional_correctness.py\", line 28, in <module>\n","    sys.exit(main())\n","             ^^^^^^\n","  File \"/content/drive/MyDrive/CS598JBR-Team-27 Drive/CS598JBR-Team-0/human-eval/human_eval/evaluate_functional_correctness.py\", line 25, in main\n","    fire.Fire(entry_point)\n","  File \"/usr/local/lib/python3.12/dist-packages/fire/core.py\", line 135, in Fire\n","    component_trace = _Fire(component, args, parsed_flag_args, context, name)\n","                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n","  File \"/usr/local/lib/python3.12/dist-packages/fire/core.py\", line 468, in _Fire\n","    component, remaining_args = _CallAndUpdateTrace(\n","                                ^^^^^^^^^^^^^^^^^^^^\n","  File \"/usr/local/lib/python3.12/dist-packages/fire/core.py\", line 684, in _CallAndUpdateTrace\n","    component = fn(*varargs, **kwargs)\n","                ^^^^^^^^^^^^^^^^^^^^^^\n","  File \"/content/drive/MyDrive/CS598JBR-Team-27 Drive/CS598JBR-Team-0/human-eval/human_eval/evaluate_functional_correctness.py\", line 20, in entry_point\n","    results = evaluate_functional_correctness(sample_file, k, n_workers, timeout, problem_file)\n","              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n","  File \"/content/drive/MyDrive/CS598JBR-Team-27 Drive/CS598JBR-Team-0/human-eval/human_eval/evaluation.py\", line 62, in evaluate_functional_correctness\n","    for sample in tqdm.tqdm(stream_jsonl(sample_file)):\n","                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n","  File \"/usr/local/lib/python3.12/dist-packages/tqdm/std.py\", line 1181, in __iter__\n","    for obj in iterable:\n","               ^^^^^^^^\n","  File \"/content/drive/MyDrive/CS598JBR-Team-27 Drive/CS598JBR-Team-0/human-eval/human_eval/data.py\", line 26, in stream_jsonl\n","    with open(filename, \"r\") as fp:\n","         ^^^^^^^^^^^^^^^^^^^\n","FileNotFoundError: [Errno 2] No such file or directory: 'instruct_prompt_processed_270992647952172920073750662102343813773.jsonl'\n"]}]},{"cell_type":"code","source":["# Use to go back\n","!pwd\n","%cd ..\n","# %cd human-eval/human_eval/\n","# !ls"],"metadata":{"colab":{"base_uri":"https://localhost:8080/"},"id":"qrSjINeqt005","executionInfo":{"status":"ok","timestamp":1758650710867,"user_tz":300,"elapsed":104,"user":{"displayName":"Maverick Espinosa","userId":"01952844037976753706"}},"outputId":"1871dc37-b1d8-4444-96be-b863e48b9244"},"execution_count":null,"outputs":[{"output_type":"stream","name":"stdout","text":["/content/drive/MyDrive/CS598JBR-Team-27 Drive/CS598JBR-Team-0/human-eval\n","/content/drive/MyDrive/CS598JBR-Team-27 Drive/CS598JBR-Team-0\n"]}]}]}